src/zed_wrapper_node.cpp

```diff
@@ -298,7 +299,7 @@ int main(int argc, char **argv) {
     nh_ns.getParam("cloud_frame_id", cloud_frame_id);
 
     // Create the ZED object
-    sl::zed::Camera *zed;
+    std::unique_ptr<sl::zed::Camera> zed;
     if (argc == 2) {
         zed = new sl::zed::Camera(argv[1]); // Argument "svo_file" in launch file
```

Comment:
It's 21:40pm for me, so I don't have the error log with me, but I'll tell you tomorrow! However, I'm not used to std::unique_ptr, but don't we need to use std::move() to set the pointed object?